.changelog/73.txt

```diff
@@ -0,0 +1,49 @@
+```release-note:breaking-change
+`client` - The `CLOUDAVENUE_ENDPOINT` environment variable has been renamed to `CLOUDAVENUE_URL`.
```

Comment:
It's not a breaking change, it's a deprecated field

Reply:
The deprecated field introduce a breaking change ?